packages/devextreme/js/__internal/scheduler/appointment_popup/m_popup.ts

```diff
@@ -111,6 +111,9 @@ export class AppointmentPopup {
       _ignorePreventScrollEventsDeprecation: true,
       onHiding: (e): void => {
         this.scheduler.focus();
+        if (editingConfig?.form?.onCanceled) {
+          editingConfig.form.onCanceled(this.form.formData);
+        }
```

Comment:
I found an issue: if I save appt, then both onSaved and onCanceled callbacks are called. Please fix it and add a test for that